<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="BtnStart">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>WorkStart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnStop">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RestStart</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>work time:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LineEditWork"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>minutes</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>rest time:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LineEditRest"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>minutes</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Adobe Arabic&quot;;</string>
       </property>
       <property name="text">
        <string>Rest Time:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="LcdMinute"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Adobe Arabic&quot;;</string>
       </property>
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="LcdSecond"/>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Adobe Arabic&quot;;</string>
       </property>
       <property name="text">
        <string>s</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>402</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
